src/cli.zig

```diff
@@ -2664,21 +2669,6 @@ pub const Command = struct {
             .PublishCommand = true,
         });
 
-        pub const always_loads_config: std.EnumArray(Tag, bool) = std.EnumArray(Tag, bool).initDefault(false, .{
-            .BuildCommand = true,
-            .TestCommand = true,
-            .InstallCommand = true,
-            .AddCommand = true,
-            .RemoveCommand = true,
-            .UpdateCommand = true,
-            .PatchCommand = true,
-            .PatchCommitCommand = true,
-            .PackageManagerCommand = true,
-            .BunxCommand = true,
-            .OutdatedCommand = true,
-            .PublishCommand = true,
-        });
-
```

Comment:
once i added the `bun run` ones, this list wasn't any different to the base `loads_config`